src/ccache.c

```diff
@@ -59,6 +59,7 @@ static const char USAGE_TEXT[] =
   "    -M, --max-size=SIZE   set maximum size of cache to SIZE (use 0 for no\n"
   "                          limit); available suffixes: k, M, G, T (decimal) and\n"
   "                          Ki, Mi, Gi, Ti (binary); default suffix: G\n"
+  "    -k, --get-config=K    get the value of the configuration key K\n"
```

Comment:
Please keep the option list in sorted order.

Reply:
I wasn't completely sure whether to sort on the short-name or the long-name, but I went for the short name.